apps/service/src/app/(blank)/long/_components/VideoPlaybackManager.tsx

```
@@ -0,0 +1,120 @@
+  if (!videoId) {
+    return (
+      <div
+        className="flex w-full items-center justify-center bg-black text-white"
+        style={{ aspectRatio: '16/9' }}
+      >
+        유효하지 않은 영상입니다.
+      </div>
+    );
+  }
+  // 시청 기록 및 비디오 메타데이터를 가져오기.
+  const { data: detailData } = useVideoDetail(Number(videoId));
```

Biome (2.4.6)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

(lint/correctness/useHookAtTopLevel)